test/unit/tasks/forum/migrate_forum_test.rb

```diff
@@ -3,8 +3,7 @@
 class Tasks::ForumTest < ActiveSupport::TestCase
 
   def test_migrate_forum
-    admin_1 = FactoryGirl.create(:simple_user, id: 1)
-    admin_2 = FactoryGirl.create(:simple_user, id: 2)
+    admin_1, admin_2 = FactoryGirl.create_list(:simple_user, 2)
```

Comment:
This is great work 👍 but IMO it is better to extract it to its own PR since it is completely unrelated to this PR and the random failure can happen to anybody any time meanwhile.